libr/core/canal.c

```diff
@@ -759,7 +759,7 @@ static bool __core_anal_fcn(RCore *core, ut64 at, ut64 from, int reftype, int de
 	const char *cc = r_anal_cc_default (core->anal);
 	if (!cc) {
 		if (r_anal_cc_once (core->anal)) {
-			eprintf ("Warning: set your favourite calling convention in `e anal.cc=?`\n");
+			R_LOG_WARN ("set your favourite calling convention in `e anal.cc=?`");
```

Comment:
revert this line as it seems to break some tests. we will look later at them
```suggestion
			eprintf ("Warning: set your favourite calling convention in `e anal.cc=?`\n");
```